foreman-selinux-enable

```diff
@@ -17,20 +18,22 @@ is_redhat_6() {
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    /usr/sbin/semanage port -E > $TMP_PORTS
+
+    # Remove previously defined elasticsearch_port_t
+    # (this can be removed in future release)
+    grep elasticsearch_port_t $TMP_PORTS | sed s/-a/-d/g >> $TMP
```

Comment:
But `semanage ... -d` can be called only when the port exists in $TMP_PORTS (which we need to call anyway for docker_port_t etc).  If it doesn't exist, we can avoid a performance penalty.